main/cmds_test.go

```diff
@@ -155,3 +157,34 @@ func Test_decodeScriptGzip(t *testing.T) {
 	require.Equal(t, info, "32;3;gzip=1")
 	require.Equal(t, s, "ls\n")
 }
+
```

Comment:
Is it possible to add more tests for example hitting the system managed identify vs a specified managed identity?

Reply:
added few cases